### ISSUE TITLE
Desktop: No longer prints to a printer

### ISSUE THREAD

**Issue by jgadsden** [OWASP/threat-dragon#212]
**Describe the bug**
When printing a report to a printer (not PDF) get error message 'Report printing failed' and no list of printers is displayed. This means that it is now not possible to print from Threat Dragon.
The console messages (using very verbose) are:
```
09:05:30.877 › Desktop Report print PDF
09:05:30.947 › Desktop Report printing failed: no valid printers available
```

This is not present on versions 1.20 and earlier, these versions work OK. This is only present on versions 1.3.0 onwards

Note that the web  application still prints to a printer OK

**Expected behavior**
A list of printers

**Environment**
 - Version: only 1.3.0 onwards, 1.20 and lower OK
 - Platform: Desktop
 - OS: MacOS, probably Windows / Linux as well
 
**To Reproduce**
1. Open a threat model
2. Click on Report
3. Click on Print

**Any additional context, screenshots, etc**

**Comment by jgadsden** [comment]
Working on this now, hope to get a fix in for version 1.4.1

**Comment by jgadsden** [comment]
This is now fixed for version 1.4.1
Needed an update on the package dependencies
